_posts/2022-03-30-cockpit-266.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 266
+author: garrett
+date: '2022-03-30'
+tags: cockpit
+slug: cockpit-266
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 266:
+
+
+## Standardize on SI-based units for data sizes and rates
```

Comment:
And when exactly did we standardise on american spellings? :)

Reply:
Since the beginning, as we work for an American company with a big bulk of its customers in the US?

Reply:
```suggestion
## Consistently use SI-based units for data sizes and rates
```

Non-regional spelling suggestion above.

Or even simpler:

```suggestion
## Use SI-based units for data sizes and rates
```

Reply:
To be clear: this comment was a joke.